### DISCUSSION TITLE
How to get number with large precision as a string?

### DISCUSSION THREAD

**Discussion by congfairy** [nlohmann/json#2723] [Q&A]
I would like to get data with large precision with nlohmann, but when I use it ,the precision has lost ,is there a function to get a number with large precision to a string?
![image](https://user-images.githubusercontent.com/13255322/115169896-e38a8280-a0f1-11eb-825f-4ff4ae4b0e21.png)
![image](https://user-images.githubusercontent.com/13255322/115169912-eab19080-a0f1-11eb-8345-bc4bfee60988.png)

**Comment by nlohmann** [comment]
Numbers are stored as `std::uint64_t`, `std::int64_t`, or `double` - whatever fits best. You have no access to the originally parsed string - this is only available via the SAX interface while parsing (https://json.nlohmann.me/features/parsing/sax_interface/).
